packages/block-editor/src/components/url-input/index.js

```diff
@@ -231,7 +231,7 @@ class URLInput extends Component {
 			! ( suggestions && suggestions.length )
 		) {
 			// Ensure the suggestions are updated with the current input value
-			this.updateSuggestions( value );
+			this.updateSuggestions( value.trim() );
```

Comment:
If the `updateSuggestions()` func is expected to always be called with trimmed `value`, it's probably better to do the trimming inside that very function. Not a blocker though, just a suggestion - feel free to 🚢 this one. Nice work!

Reply:
IMHO, it's only duplicated once now. When we need `trimming` once more, it's strikeout and let's refactor. 

I cannot think of a case that doesn't need trimming. But we don't know the future.